src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -80,7 +88,8 @@ const LaunchAnalysisModal = ({
     [type === processAll, () => entityMetadata[rootEntityType].count],
     [type === processAllAsSet, () => 1],
     [type === chooseSetComponents, () => 1],
-    [type === processMergedSet, () => _.flow(mergeSets, _.uniqBy('entityName'))(selectedEntities).length]
+    [type === processMergedSet, () => _.flow(mergeSets, _.uniqBy('entityName'))(selectedEntities).length],
+    [type === processSnapshotTable, () => 1] //TODO: could be table rowCount but rowCount to workflow conversion is currently unclear
```

Comment:
Using `rowCount` from the snapshot table details is probably the right move in the end, but these submissions were spawning 0 workflows (but still writing outputs). I've followed the examples in the Rawls unit tests but I think there may be something missing either here or there.